gprofiler/main.py

```diff
@@ -281,6 +304,13 @@ def parse_cmd_args():
 
     parser.add_argument("-v", "--verbose", action="store_true", default=False, dest="verbose")
     parser.add_argument("--log-file", action="store", type=str, dest="log_file", default=DEFAULT_LOG_FILE)
+    parser.add_argument(
+        "--disable-container-names",
+        action="store_true",
+        dest="disable_container_names",
+        default=False,
+        help="The gProfiler won't gather the container names of processes that run in " "containers",
```

Comment:
```suggestion
        help="gProfiler won't gather the container names of processes that run in " "containers",
```

Also, the string is fucked up (2 stray `"`)